appveyor.yml

```diff
@@ -1,58 +1,58 @@
-configuration: Debug
-image: Visual Studio 2019
-
-version: '{build}'
-
-init:
-- ps: Update-AppveyorBuild -Version "$($env:ospsuite_version).$($env:appveyor_build_version)"
-
-clone_depth: 1 
-
-dotnet_csproj:
-  patch: true
-  file: '**\*.csproj'
-  version: '$(ospsuite_version).$(build_number)'
-  assembly_version: '$(ospsuite_version).$(build_number)'
-  file_version: '$(ospsuite_version).$(build_number)'
-  informational_version: '$(ospsuite_version).$(build_number)'
-
-before_build:
-  - nuget sources add -name bddhelper -source https://ci.appveyor.com/nuget/ospsuite-bddhelper 
-  - nuget sources add -name utility -source https://ci.appveyor.com/nuget/ospsuite-utility 
-  - nuget sources add -name serializer -source https://ci.appveyor.com/nuget/ospsuite-serializer 
-  - nuget sources add -name texreporting -source https://ci.appveyor.com/nuget/ospsuite-texreporting 
-  - nuget sources add -name databinding -source https://ci.appveyor.com/nuget/ospsuite-databinding 
-  - nuget sources add -name databinding-devexpress -source https://ci.appveyor.com/nuget/ospsuite-databinding-devexpress 
-  - nuget sources add -name funcparser -source https://ci.appveyor.com/nuget/ospsuite-funcparser 
-  - nuget sources add -name simmodel -source https://ci.appveyor.com/nuget/ospsuite-simmodel
-  - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes
-  - nuget sources add -name core -source https://ci.appveyor.com/nuget/ospsuite-core
-  - nuget restore
-
-build:
-  verbosity: minimal
-  project: MoBi.sln
-
-test:
-  assemblies:
-    - MoBi.Tests.dll
-    - MoBi.UI.Tests.dll
-
-  categories:
-    except:
-      - IntegrationTests
-
-skip_commits:
-  files:
-    - '*.md'
-
-skip_branch_with_pr: true
-
-skip_tags: true
-
-branches:
-  only:
-    - develop
-
-pull_requests: 
-  do_not_increment_build_number: true     
+configuration: Debug
+image: Visual Studio 2019
+
+version: '{build}'
+
+init:
+- ps: Update-AppveyorBuild -Version "11.3.$($env:appveyor_build_version)"

```

Comment:
not sure if we need to change this one. It's not were the build are created from